pnpm-lock.yaml

```diff
@@ -10862,7 +10235,7 @@ packages:
       bundle-require: 4.0.2(esbuild@0.19.12)
       cac: 6.7.14
       chokidar: 3.6.0
-      debug: 4.3.4(supports-color@8.1.1)
```

Comment:
There may be something here that might be worth keeping around for nicer output 🌈